### PR TITLE
New package: FreakinSoftMania.WinUEFI version v1.9.2

### PR DESCRIPTION
**Pull request by Minionguyjpro**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111690&drop=dogfoodAlpha

### CODE REVIEW

**Comment by Minionguyjpro** [comment]
> 

@microsoft-github-policy-service agree [company="{FreakinSoftMania"]

**Comment by Minionguyjpro** [comment]
@microsoft-github-policy-service agree

**Comment by Minionguyjpro** [comment]
What is wrong with the executable?

**Comment by BrandonWanHuanSheng** [comment]
Hmnm, I might guess could be dependencies or portable package returned a non zero code.

**Comment by Minionguyjpro** [comment]
> Hmnm, I might guess could be dependencies or portable package returned a non zero code.

Wait, what do you mean? How would I be able to fix that?

**Comment by BrandonWanHuanSheng** [comment]
> > Hmnm, I might guess could be dependencies or portable package returned a non zero code.
> 
> Wait, what do you mean? How would I be able to fix that?

Someone assigned on the pull request will add the waiver label.

**Comment by stephengillie** [comment]
Hi @Minionguyjpro,

Unfortunately, the installer did not pass a Defender scan:
image

Please virus scan your compiling or packaging systems to be sure they are uninfected. This might be a false positive, and I can recheck tomorrow, after the Defender team has updated its antivirus version.

**Comment by Minionguyjpro** [comment]
> Hi @Minionguyjpro,
> 
> Unfortunately, the installer did not pass a Defender scan:
> image
> 
> Please virus scan your compiling or packaging systems to be sure they are uninfected. This might be a false positive, and I can recheck tomorrow, after the Defender team has updated its antivirus version. 

I think this is a false positive. I used it even myself several times. It might be due to what the program does, that it thinks that it is a virus.

**Comment by Minionguyjpro** [comment]
> Hi @Minionguyjpro,
> 
> Unfortunately, the installer did not pass a Defender scan: image
> 
> Please virus scan your compiling or packaging systems to be sure they are uninfected. This might be a false positive, and I can recheck tomorrow, after the Defender team has updated its antivirus version.

Looks good to me:
image

**Comment by Minionguyjpro** [comment]
Ehm, why does it take so long?

**Comment by Minionguyjpro** [comment]
@stephengillie, did you check again? Also, what can I do to prevent this in the future? I am pretty sure my compilers are clean. I heard about signing the program, but from what I saw such a code signing certificate costs money, which I do not think that fits into open source software though.

**Comment by stephengillie** [comment]
> @stephengillie, did you check again? Also, what can I do to prevent this in the future?

I lost track of this PR, sorry.

**Comment by Minionguyjpro** [comment]
So what does that mean??? It will not be added, because no one cares? It is safe, believe me. Deeply scan, but I think you won't find any harm.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> ##[error] Installer https://github.com/FreakinSoftMania/WinUEFI/releases/download/v1.9.2/WinUEFI-setup.exe
Scan request id: 'c6fb4620-0cd4-40e8-8a33-78272c5a0115'
Error information: BlockingDetectionFound
##[error] Result: Failed

Scanning was prevented due to an incomplete download, after almost 8 hours of downloading.

**Comment by Minionguyjpro** [comment]
Why did it take so long? This can't be an issue at my side as it is just an
URL from GitHub.

Op vr 21 jul. 2023 16:34 schreef Stephen Gillie ***@***.***>:

**Comment by Minionguyjpro** [comment]
> Hello @Minionguyjpro,
> 
> The package manager bot determined there was an issue with one of the installers listed in the url field, and cannot continue.
> 
> The application included in this pull request failed to pass the [Installers Scan](https://docs.microsoft.com/en-us/windows/package-manager/package/winget-validation) test. This test is designed to ensure that the application installs on all environments without warnings. For more details on this error, see [Binary validation errors](https://docs.microsoft.com/en-us/windows/package-manager/package/binary-validation-errors). Please check to ensure the installer URL is correct and update the URL and the Hash if a change is made.
> 
> Template: msftbot/validationError/installers/binaryValidation

I just tested, and the hash is still the same! This must go wrong somewhere here.

**Comment by Minionguyjpro** [comment]
> Hi @Minionguyjpro,
> 
> > I just tested, and the hash is still the same! This must go wrong somewhere here.
> 
> Sorry, that's our bot's default suggestion. The package is still blocked from download by our scans. 

Okay. What can I do about it?

**Comment by Minionguyjpro** [comment]
> Hello @Minionguyjpro,
> 
> The package manager bot determined there was an issue with one of the installers listed in the url field, and cannot continue.
> 
> The application included in this pull request failed to pass the [Installers Scan](https://docs.microsoft.com/en-us/windows/package-manager/package/winget-validation) test. This test is designed to ensure that the application installs on all environments without warnings. For more details on this error, see [Binary validation errors](https://docs.microsoft.com/en-us/windows/package-manager/package/binary-validation-errors). Please check to ensure the installer URL is correct and update the URL and the Hash if a change is made.
> 
> Template: msftbot/validationError/installers/binaryValidation

I already tested. There is nothing wrong with it!

**Comment by stephengillie** [comment]
Hi @Minionguyjpro,

Sorry for losing track of this PR, and also for the unclear answers. 
- We scan packages in 2 main ways during validation. 
- This PR is stuck in the first scan step. 
- It looks like it's stuck because the URL doesn't allow automatic download, or is blocking in another way. This would have to be modified by the ISV (Independent Software Vendor).
- I thought I could manually scan this at the start of July and approve that way, but since then I've learned that my manual scans are the second scan step. 
- So I'm not able to manually validate this PR, sorry. We'll have to wait for Automatic Validation to get past this step. If it fails on the next step, then I can manually validate. 

Does this help? Please @ me in any reply.

**Comment by Minionguyjpro** [comment]
Okay. What could I do about it? It is just a direct download link for a
GitHub asset (setup executable).

Op wo 2 aug. 2023 18:33 schreef Stephen Gillie ***@***.***>:

File: manifests/f/FreakinSoftMania/WinUEFI/v1.9.2/FreakinSoftMania.WinUEFI.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: FreakinSoftMania.WinUEFI
+PackageVersion: v1.9.2
+PackageLocale: en-US
+Publisher: FreakinSoftMania
```
Comment (ItzLevvie) [inline manifests/f/FreakinSoftMania/WinUEFI/v1.9.2/FreakinSoftMania.WinUEFI.locale.en-US.yaml:7]:
```suggestion
Publisher: Minionguyjpro
```
This must match Control Panel.

File: manifests/f/FreakinSoftMania/WinUEFI/v1.9.2/FreakinSoftMania.WinUEFI.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: FreakinSoftMania.WinUEFI
+PackageVersion: v1.9.2
```
Comment (ItzLevvie) [inline manifests/f/FreakinSoftMania/WinUEFI/v1.9.2/FreakinSoftMania.WinUEFI.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.9.2
```
This must match Control Panel.

File: manifests/f/FreakinSoftMania/WinUEFI/v1.9.2/FreakinSoftMania.WinUEFI.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: FreakinSoftMania.WinUEFI
+PackageVersion: v1.9.2
```
Comment (ItzLevvie) [inline manifests/f/FreakinSoftMania/WinUEFI/v1.9.2/FreakinSoftMania.WinUEFI.installer.yaml:5]:
```suggestion
PackageVersion: 1.9.2
```
This must match Control Panel.

File: manifests/f/FreakinSoftMania/WinUEFI/v1.9.2/FreakinSoftMania.WinUEFI.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: FreakinSoftMania.WinUEFI
+PackageVersion: v1.9.2
```
Comment (ItzLevvie) [inline manifests/f/FreakinSoftMania/WinUEFI/v1.9.2/FreakinSoftMania.WinUEFI.yaml:5]:
```suggestion
PackageVersion: 1.9.2
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/f/FreakinSoftMania/WinUEFI/v1.9.2/FreakinSoftMania.WinUEFI.locale.en-US.yaml (hunk 1 above)
Comment (Minionguyjpro) [reply manifests/f/FreakinSoftMania/WinUEFI/v1.9.2/FreakinSoftMania.WinUEFI.locale.en-US.yaml:7]: What do you mean, control panel? I changed this for a reason. I rebranded it from myself to that GutHub organization.

File: manifests/f/FreakinSoftMania/WinUEFI/v1.9.2/FreakinSoftMania.WinUEFI.locale.en-US.yaml (hunk 2 above)
Comment (Minionguyjpro) [reply manifests/f/FreakinSoftMania/WinUEFI/v1.9.2/FreakinSoftMania.WinUEFI.locale.en-US.yaml:5]: I can change this indeed. Will do!

File: manifests/f/FreakinSoftMania/WinUEFI/v1.9.2/FreakinSoftMania.WinUEFI.locale.en-US.yaml (hunk 1 above)
Comment (Minionguyjpro) [reply manifests/f/FreakinSoftMania/WinUEFI/v1.9.2/FreakinSoftMania.WinUEFI.locale.en-US.yaml:7]: What should I do? Make a new release first with the updated publisher? I changed it in the setup script (ISS). I know this is also responsible for making the control panel entry at programs. I woll do that and change it a little again after that.

**Comment by Minionguyjpro** [comment]
I have made some changes here and made a new PR with a new version with the fixed changes. It should be correct now. If you don't mind @ItzLevvie and @stephengillie, please take a look at the new PR #115062. Going to close this one.

**Comment by stephengillie** [comment]
> I have made some changes here and made a new PR with a new version with the fixed changes. It should be correct now. If you don't mind @ItzLevvie and @stephengillie, please take a look at the new PR #115062. Going to close this one. Otherwise it might have caused issue with the branch name and an update, while this was not even merged yet so.

Hi @Minionguyjpro,

I'm just seeing this request, sorry. Please feel free to remake the branch and submit the PR again, and I'd be happy to review it.

**Pull request closed without merging**